includes/wp-admin/class-user-settings-fields.php

```diff
@@ -380,4 +388,20 @@ public static function blocked_keywords_callback() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Hide Social Graph field callback.
+	 */
+	public static function hide_followers_callback() {
+		$hide_followers = \get_user_option( 'activitypub_hide_social_graph', \get_current_user_id() );
+		?>
+		<label>
+			<input type="checkbox" name="activitypub_hide_social_graph" id="activitypub_hide_social_graph" value="1" <?php \checked( '1', $hide_followers ); ?> />
+			<?php \esc_html_e( 'Hide Followers and Following on Profile', 'activitypub' ); ?>
+		</label>
+		<p class="description">
+			<?php esc_html_e( 'By default, these lists are public. Hiding them removes them from your profile, but people you follow will still see that you follow them.', 'activitypub' ); ?>
```

Comment:
```suggestion
			<?php esc_html_e( 'People you follow will still see that you follow them.', 'activitypub' ); ?>
```
I worry that the current phrasing could make folks feel like it's something that _should_ be hidden, when that's not the case. So rather than "warning" it's currently public, maybe we could just limit it to the expectation setting around followers still able to see the relationship?